src/modules/layerManager/components/LayerItem.js

```diff
@@ -218,13 +219,26 @@ export class LayerItem extends AbstractMvuContentPanel {
 			}
 		};
 
+		const getFeatureCountBadge = (featureCount, layerState) => {
+			return (featureCount || isNumber(featureCount)) && layerState !== LayerState.LOADING
```

Comment:
`return isNumber(featureCount) && layerState !== LayerState.LOADING` should be sufficient